setup.py

```diff
@@ -149,6 +149,7 @@ def run(self):
                 '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON',
                 '-DONNX_NAMESPACE={}'.format(ONNX_NAMESPACE),
                 '-DPY_EXT_SUFFIX={}'.format(sysconfig.get_config_var('EXT_SUFFIX') or ''),
+                '-DENABLE_GTESTS=ON'
```

Comment:
python build should not build anything that python side doesn't need (in this case cpp tests are not usable from python side).

Reply:
Good point.. I ll change this to enabling UT in CI instead